test/wine_o_inventory_api/businesses_test.exs

```diff
@@ -0,0 +1,14 @@
+defmodule WineOInventoryApi.BusinessesTest do
+    use WineOInventoryApi.DataCase
+
+    alias WineOInventoryApi.Businesses
+
+    describe "stores" do
+
+        test "list_wines_with_rating_from_winery/3 returns all wines with given rating from given winery" do
+            assert Businesses.list_wines_with_rating_from_winery("some store", 3, "some winery") == []
```

Comment:
I would expect some more tests, to help provide some documentation about what this query is supposed to do.